spec/services/team_move_service_spec.rb

```diff
@@ -35,8 +35,17 @@
         team = TeamMoveService.new( bu, target_dir)
         expect(team.new_unit.directorate).to eq target_dir
       end
+      it 'Moves team user_roles to the new team, and removes them from the original team' do
+
+        tur = TeamsUsersRole.create user: create(:user),
+                                    team: bu,
+                                    role: 'manager'
+
+        team = TeamMoveService.new( bu, target_dir)
+        expect(team.new_unit.user_roles.first).to eq tur
+        expect(bu.user_roles.first).to eq nil
+      end
 
```

Comment:
Formatting is a bit off, i.e. 

- test name string should not be capitalised (moves/Moves)
- should be newline Line 38
- no new line on Line 40
- no space inopening bracket Line 44
- prefer human readable variable names rather than abbr e.g `tur/team_user_role`
- can use `be_nil` rather than `eq nil` but that's personal preference imo

i personally prefer formatting of multi-line statements as follows:

```
team_user_role = TeamsUsersRole.create(
  user: create(:user),
  team: bu,
  role: 'manager'
)
```
That helps avoid random right-alignment issues and keeps things tidy/easy to read in column format which works nicely on widescreens.

Regarding execution, I notice the `TeamMoveService` performs the required actions on construction, rather than requiring an additional method call e.g. `TeamMoveService.new(bu, target_dir).call` or similar. Is that a deliberate decision? (Many of the services require .call with appropriate flags set).

Reply:
Thanks Mo, comments on formatting noted. 
Since the service does one thing only, it made sense to me to do the work on construction rather than with .call + possible params. Simplest thing possible is my preference.

Reply:
Regarding atomicity, that's in another card https://dsdmoj.atlassian.net/browse/CT-2591?atlOrigin=eyJpIjoiOTAzZjg0MzUzMThjNDA0MjhmZTA3MjQ5ZmEwMDQ5ZWUiLCJwIjoiaiJ9